### DISCUSSION TITLE
Cache synchronization

### DISCUSSION THREAD

**Discussion by kalfonh** [MichaCo/CacheManager#371] [Q&A] [answered]
Hello,
Is there a way to use Redis as backplane (for multiple cache synchronization) but subscribe to only specific channels? 
If so, could you publish some exemples?

Here is my configuration:

```
 var config = ConfigurationBuilder.BuildConfiguration(settings =>
        {
            settings
                 .WithJsonSerializer()
                 .WithMicrosoftMemoryCacheHandle("in-memory")
                 .And
                 .WithRedisConfiguration("redis", cfg =>
                 {
                     cfg
                         .WithAllowAdmin()
                         .WithEndpoint("my-endpoint", 6379)
                         .WithPassword("***********")
                         .WithSsl()
                         .EnableKeyspaceEvents()
                         .Build();
                 })
                 .WithRedisBackplane("redis")
                 .WithMaxRetries(300)
                 .WithRetryTimeout(1000)
                 .WithRedisCacheHandle("redis", true)
                     .WithExpiration(ExpirationMode.Absolute, TimeSpan.FromMinutes(10))
                     .EnablePerformanceCounters()
                     .EnableStatistics()
                 .Build();
        });
```

**Comment by MichaConrad** [comment, marked as answer]
Not 100% sure if that's what you mean, but `WithRedisBackplane` has an overload to specify the channel name it should use.

**Reply by kalfonh** [reply]
Thank you for replying so fast.

For clarification, let say I have multiples applications that use the same distributed cache (redis) but each application wants to subscribe to specifics channels, in order to use the pub/sub mechanism implemented by backplane, should it look like this?

App A:
 `
.WithRedisBackplane("redis", "channelY")
.WithRedisBackplane("redis", "channelX")
`

App B:
 `
.WithRedisBackplane("redis", "channelX")
.WithRedisBackplane("redis", "channelZ")
`

Or am I missing something?

**Reply by MichaConrad** [reply]
yep, if you want to prevent both apps sending each other messages, which might be irrelevant, just use different channel names.
